webapp/src/marked.tsx

```diff
@@ -384,7 +384,7 @@ export class MarkedContent extends data.Component<MarkedContentProps, MarkedCont
     // Renders inline blocks, such as "||controller: Controller||".
     private renderInlineBlocks(content: HTMLElement) {
         const { parent } = this.props;
-        const hasCategories = !parent.state.tutorialOptions?.metadata?.flyoutOnly;
+        const hasCategories = !parent.state.tutorialOptions?.metadata?.flyoutOnly && !parent.state.tutorialOptions?.metadata?.hideToolbox;
```

Comment:
Thanks for making the change, I think this might fail in the blocks scenario now, though, since you said that the `hideToolbox` flag is only being used in Monaco 😅. 
```suggestion
        const hasCategories = !parent.state.tutorialOptions?.metadata?.flyoutOnly || !parent.state.tutorialOptions?.metadata?.hideToolbox;
```

Reply:
I think it only **has** categories when both of these checks are false, right?

Reply:
Can confirm, blocks look as expected (I think):
![image](https://github.com/user-attachments/assets/42cb9d63-59f0-4982-b9a7-fd2fa4e39687)

Reply:
Oh you are totally right. Sorry about that!

Reply:
Yep, that looks good!